### PR TITLE
JavaScript Job Posting 

### PR DESCRIPTION
**Pull request by patriciarealini**
## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._
seal

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._
https://trello.com/c/lGqa2KUz/65-hire-javascript-developer

## Description
_The commit messages say what you did; this should explain why and/or how._
- [x] Description is in Trello card

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

## Show me
_Provide screenshots/animated gifs/videos if necessary._

### CODE REVIEW

**Comment by patriciarealini** [comment]
ready for review!

**Comment by patriciarealini** [comment]
[Hire Javascript Developer](https://trello.com/c/lGqa2KUz/65-hire-javascript-developer)

**Review by mirisuzanne — APPROVED** [review]
lgtm!

**Review by stacyk — APPROVED** [review]
Looks good.

File: content/jobs/js2021.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+position: open
+banner: JavaScript Developer
+sub: Leading our front-end development process
```
Comment (jgerigmeyer) [inline content/jobs/js2021.md:4]: At 8 people, are we all "leading"? 🤔

File: content/_data/site.yaml [hunk 2]
```diff
@@ -19,4 +19,5 @@ nav:
   - url: /open-source/
   - url: /blog/
   - url: /birds/
+  - url: /jobs/
```
Comment (jgerigmeyer) [inline content/_data/site.yaml:22]: We might want to review/edit the OddBirding Guide linked from this page? There are a few items there that seem out of date, esp during COVID.

**Review by jgerigmeyer — COMMENTED** [review]
Given that we said this wasn't urgent, I would've appreciated a chance to review (during work hours) before this went live, but regardless it looks good to me for now and we can make any changes post-merge. 👍🏻

**Comment by patriciarealini** [comment]
[JavaScript Job Posting](https://trello.com/c/0WDHYuWe/127-javascript-job-posting)

File: content/jobs/js2021.md (hunk 1 above)
Comment (patriciarealini) [reply content/jobs/js2021.md:4]: I made a revision suggestion to change it to `Collaborating on our front-end development process` here: https://github.com/oddbird/oddleventy/pull/134/files

File: content/_data/site.yaml (hunk 2 above)
Comment (patriciarealini) [reply content/_data/site.yaml:22]: I made a revision suggestion to add `(For the safety of our birds & our communities, due to the COVID pandemic our retreats are currently being conducted remotely.)` to the Oddbirding Guide here: https://github.com/oddbird/oddleventy/pull/134/files

**Pull request merged**